Automattic/pocket-casts-android
# Add small device support to Top Show EoY story

## Description

This PR adds better support for small devices when displaying Top Show story. The cutout shape is now calculated in a way that it's diagonal is equal to the available space's height so it doesn't go below the text.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3070_23365

## Testing Instructions

1. Start PB24.
2. Go to the Top Show story.
3. Verify the UI.
4. Share the story and make sure that the screenshot looks correctly.

## Screenshots or Screencast 

| Regular | Small |
| - | - |
| ![large](https://github.com/user-attachments/assets/e0ca1112-682a-48c1-80ce-c46d6ca52301) | ![smol](https://github.com/user-attachments/assets/0bdf6141-b477-4ca8-8129-3dd5a9bf3c1e) |

## Checklist
